src/main/java/de/doubleslash/keeptime/viewpopup/ViewControllerPopup.java

```diff
@@ -115,6 +107,15 @@ private void initialize() {
 
       Interval.registerCallBack(() -> projectsListViewController.tick());
 
+      projectsListViewController = new ProjectsListViewController(model, controller, stage, projectListView,
```

Comment:
stage is null here, resulting in npe with popupstage when "change with time" is used.
we may still need the 2nd initialize here?